File: SDK/DotNet/Uloq.SDK/Uloq.SDK/Models/OutcomeObject.cs
using System;
using System.Collections.Generic;
using System.Runtime.CompilerServices;
using System.Text;

namespace Uloq.SDK.Models
{
    public class OutcomeObject<T>
    {
        public int StatusCode { get; set; } = 200;
        public bool Success { get; set; } = true;
        public string Message { get; set; } = "";
        public T Result { get; set; } = default;
    }

    public class OutcomeObject
    {
        public int StatusCode { get; set; } = 200;
        public bool Success { get; set; } = true;
        public string Message { get; set; } = "";
        public object Result { get; set; } = null;
    }
}

File: SDK/Python/uloq_sdk/my_module/models/OutcomeObject.py

from typing import TypeVar, Generic, Optional

T = TypeVar('T')

class OutcomeObject(Generic[T]):
    def __init__(self, success: bool, result: Optional[T] = None, message: Optional[str] = None):
        self.success = success
        self.result = result
        self.message = message
